jenkins/opensearch-dashboards/integ-test.jenkinsfile

```diff
@@ -116,8 +116,6 @@ pipeline {
                     env.artifactPath = buildManifestObj.getArtifactRoot(BUILD_JOB_NAME, buildId)
                     env.artifactPathOpenSearch = buildManifestObjOpenSearch.getArtifactRoot(BUILD_JOB_NAME_OPENSEARCH, buildIdOpenSearch)
                     env.AGENT_LABEL = agent_nodes["${env.platform}_${architecture}"]
-                    env.updateGithubIssues = (params.UPDATE_GITHUB_ISSUES && (env.distribution == 'tar'))
```

Comment:
Does the auto cut test issue can group for different distributions now? I think we used to update issue on `tar` to avoid duplicate issue cut.

Reply:
Yes we modified the issue for one generic issue on in https://github.com/opensearch-project/opensearch-build/pull/4400

Reply:
Here to be exact: https://github.com/opensearch-project/opensearch-build/pull/4400/files#diff-b2319ed0b0cd8622aa148bf51b2d2c3e8619786f9703ff9a757870b7894832b4R274